t3f/decompositions_test.py

```diff
@@ -62,6 +62,43 @@ def testTTVector(self):
     with self.test_session():
       self.assertAllClose(vec, ops.full(tt_vec).eval())
 
+ 
+  def testTTCompositeRank(self):
+    # Test if a composite rank (list of ranks) can be used for decomposition
```

Comment:
Dot in the end of the sentence.
"# Test if a composite rank (list of ranks) can be used for decomposition."